pluto_build/bin/config.sh

```diff
@@ -146,10 +146,8 @@ function import_qaqc {
     echo "✅ $name.sql exists in cache"
   else
     echo "🛠 $name.sql doesn't exists in cache, downloading ..."
-    echo "{$url}/{$name.sql}"
```

Comment:
I do find this print out sometimes helpful with getting a visual on url being curled e.g. when the url is wrong or the version of the source is wrong with debugging purposes in mind.

Reply:
I do think the url itself is useful as well but open to others' opinions on this if it seems excessive to everyone else.

Reply:
I doubt anyone else will have an opinion on this lol, I'll add back that echo statement lol

Reply:
Ok Max is on my side I'm taking it back out